File: mRemoteV1/Connection/Connection.Protocol.RAW.vb
Namespace Connection

    Namespace Protocol
        Public Class RAW
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.raw
            End Sub

            Public Enum Defaults
                None = 0
                Port = 23
            End Enum
        End Class
    End Namespace

End Namespace

File: mRemoteV1/CS/Connection/Connection.Protocol.RAW.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;
// End of VB project level imports


namespace mRemoteNG.Connection.Protocol
{
	public class RAW : Connection.Protocol.PuttyBase
			{
				
				public RAW()
				{
					this.PuttyProtocol = Putty_Protocol.raw;
				}
				
				public enum Defaults
				{
					Port = 23
				}
			}
}
